### ISSUE TITLE
The Status of the Speech Recognition Example

### ISSUE THREAD

**Issue by sbodenstein** [apache/incubator-mxnet#7796]
@Soonhwan-Kwon: we would like to make use of your speech recognition example. We had some questions:

1. I don't see any use of the more efficient cuDNN RNN operator being used. Was there a reason for this? (particularly because you quote a training time of around 900 hours).

2. Are there any significant differences between the speech recognition example in MXNet and the one in here: https://github.com/samsungsds-rnd/deepspeech.mxnet 

Thanks!

**Comment by Soonhwan-Kwon** [comment]
1. It was because it uses batch normalization with rnn. It seemed that cuDNN RNN can't do batchnorm when we made example. 
2. There is no significant difference between
samsunsds repository and example.

**Comment by szha** [comment]
This issue is closed due to lack of activity in the last 90 days. Feel free to ping me to reopen if this is still an active issue. Thanks!
Also, do please check out our [forum](https://discuss.mxnet.io/) (and [Chinese version](https://discuss.gluon.ai/)) for general "how-to" questions.
